.github/workflows/ci.yml

```diff
@@ -31,25 +31,25 @@ jobs:
         fi
       shell: bash
 
-  lint:
-    name: Lint Check
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v3
-
-      - name: Setup Go
-        uses: actions/setup-go@v4
-        with:
-          go-version: ${{env.GO_VERSION}}
-
-      - name: Generate API
-        uses: ./.github/workflows/protobuf
-
-      - name: golangci-lint
-        uses: golangci/golangci-lint-action@v3
-        with:
-          version: v1.54.2
+  # lint:
```

Comment:
I turned off lint checking in ci because I thought that refactor to the kv api might break other features that depend on it, and lint wouldn't pass at that point.
But in reality it seems unnecessary, and I've turned lint checking back on.